src/components/TodoList/TodoList.tsx

```diff
@@ -1,12 +1,26 @@
 /* eslint-disable */
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { filteredTodos } from '../../features/filter';
+import { useTypedSelector } from '../../app/hooks';
+import { setCurrentTodo } from '../../features/currentTodo';
 
 export const TodoList: React.FC = () => {
+  const todos = useTypedSelector(state => state.todos);
+  const filter = useTypedSelector(state => state.filter);
+  const currentTodo = useTypedSelector(state => state.currentTodo);
+
+  const dispatch = useDispatch();
+
+  const filtredArrayOfTodos = filteredTodos(todos, filter);
```

Comment:
Verify that the `filteredTodos` function is correctly implemented and imported from the `filter` feature. This function is essential for filtering the todos list based on the current filter state.